Jenkinsfile_nightly

```diff
@@ -49,23 +84,55 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
     ]
 }
 
+def setAatEnvVars() {
+    env.PREVIEW_BRANCH_WITH_LOCAL_CCD = false;
+    env.TEST_E2E_URL_WEB = "https://manage-case.aat.platform.hmcts.net/"
+    env.TEST_E2E_API_URI = "http://sscs-tribunals-api-aat.service.core-compute-aat.internal"
+    env.CCD_API_URL = "http://ccd-data-store-api-aat.service.core-compute-aat.internal"
+    env.CORE_CASE_DATA_API_URL = "http://ccd-data-store-api-aat.service.core-compute-aat.internal"
+    env.CCD_GW_API = "https://gateway-ccd.aat.platform.hmcts.net"
+    env.CASE_DOCUMENT_AM_URL = "http://ccd-case-document-am-api-aat.service.core-compute-aat.internal"
+    env.TEST_DOCUMENT_MANAGEMENT_URL = 'http://dm-store-aat.service.core-compute-aat.internal'
+    env.DOCUMENT_MANAGEMENT_URL = 'http://dm-store-aat.service.core-compute-aat.internal'
+    env.ENVIRONMENT="aat"
+}
+
 withNightlyPipeline(type, product, component) {
-    env.TEST_URL = params.URL_TO_TEST
+    env.TEST_URL = params.API_URL_TO_TEST
     env.POST_HEARINGS_FEATURE = false
     env.Rules = params.SecurityRules
-    env.TEST_DOCUMENT_MANAGEMENT_URL = 'http://dm-store-aat.service.core-compute-aat.internal'
-    env.DOCUMENT_MANAGEMENT_URL = 'http://dm-store-aat.service.core-compute-aat.internal'
-    env.CORE_CASE_DATA_API_URL = "http://ccd-data-store-api-aat.service.core-compute-aat.internal"
-    env.CASE_DOCUMENT_AM_URL = "http://ccd-case-document-am-api-aat.service.core-compute-aat.internal"
+    setAatEnvVars()
+    env.TEST_E2E_URL_WEB = params.UI_URL_TO_TEST
 
     loadVaultSecrets(secrets)
     enableSlackNotifications('#sscs-tech')
     enableFortifyScan()
-    enableSecurityScan()
```

Comment:
Why has be removed the security scan..?Is it because of legacy...

Reply:
@pats-john yes correct, we have moved to fortify to ensure security issues are captured as a service.